src/cps.hpp

```diff
@@ -4,11 +4,30 @@
 
 #ifndef CS372_CPS_CPS_HPP
 #define CS372_CPS_CPS_HPP
+#include <iostream>
 
+class Shape {
+    public:
+    ~Shape();
+    Shape();
+    virtual std::iostream draw()=0;  
+};
+class Polygon: Shape{
```

Comment:
Tried using this code, looks good but I needed to add public to each of these in order to be able to use them

```c++
class Polygon: public Shape {};
```

Reply:
see: https://github.com/Jlrine2/CS372-CPS/pull/5